README.rst

```diff
@@ -18,14 +20,53 @@ Usage:
     cmake --build builddir --parallel <number of cores>
     cmake --build builddir --target install
 
-Using this code via docker:
-
+Using this code via docker
+##########################
 ::
 
     docker pull votca/votca
     docker run -it votca/votca /bin/bash
 
-For further details see:
+
+
+Performance advice
```

Comment:
Should this go in the install guide ( https://www.votca.org/INSTALL.html ) instead of the README?

Reply:
I thought it was a bit more on the nose here. Not sure if it doesn't get lost in the install guide?

If we put it there, we should definitely link it directly from here.

Reply:
I think it would be better to link it here, as the info is very specific.